<?xml version="1.0" encoding="ISO-8859-1"?>
<TestCase name="tst_Assert">
<Environment>
    <QtVersion>4.6.3</QtVersion>
    <QTestVersion>4.6.3</QTestVersion>
</Environment>
<TestFunction name="initTestCase">
<Incident type="pass" file="" line="0" />
</TestFunction>
<TestFunction name="testNumber1">
<Incident type="pass" file="" line="0" />
</TestFunction>
<TestFunction name="testNumber2">
<Message type="qfatal" file="" line="0">
    <Description><![CDATA[ASSERT: "false" in file /local/user_builds/qt/4.6/tests/auto/selftests/assert/tst_assert.cpp, line 62]]></Description>
</Message>
<Incident type="fail" file="Unknown file" line="0">
    <Description><![CDATA[Received a fatal error.]]></Description>
</Incident>
</TestFunction>
</TestCase>
